doc/en/how-to/fixtures.rst

```diff
@@ -1267,7 +1267,7 @@ So let's just do another run:
     ========================= short test summary info ==========================
     FAILED test_module.py::test_ehlo[smtp.gmail.com] - assert 0
     FAILED test_module.py::test_noop[smtp.gmail.com] - assert 0
-    FAILED test_module.py::test_ehlo[mail.python.org] - AssertionError: asser...
+    FAILED test_module.py::test_ehlo[mail.python.org] - AssertionError: assert...
```

Comment:
this is not a typo, this is terminal output, same for the rest of the similar ones